x-pack/plugins/infra/public/components/asset_details/content/content.tsx

```diff
@@ -30,25 +29,26 @@ export const Content = ({
     onTabsStateChange(state);
   };
 
+  const timeRangeTs = toTimestampRange(timeRange);
   return (
```

Comment:
I didn't change the components here to use the new context, to avoid making this PR bigger than it already is. Might be a good idea to do that in the future